packages/expo-media-library/src/MediaLibrary.ts

```diff
@@ -145,6 +145,63 @@ export type AssetInfo = Asset & {
    * @platform ios
    */
   orientation?: number;
+  /**
+   * This field is available if the type is an image, and the subtype is livePhoto
+   * Whether the asset is stored on the network (iCloud on iOS).
+   * @platform ios
+   */
+  pairedVideoAsset?: PairedVideoAsset | null;
+};
+
+/**
+ * Represents a video component of a Live Photo on iOS. This is the paired video asset that
+ * accompanies a Live Photo's still image, containing the motion content.
+ */
+export type PairedVideoAsset = {
+  /**
+   * Local file URI to the extracted video component of the Live Photo.
+   * Can be used with expo-live-photo
+   */
+  uri: string;
+  /**
+   * The unique identifier of the original Live Photo asset in the iOS Photos library.
+   * Can be used by [expo-media-library](./media-library) to manage the picked asset.
+   * @platform ios
+   */
+  assetId?: string | null;
+  /**
+   * Width of the image or video
+   */
+  width: number;
+  /**
+   * Height of the image or video
+   */
+  height: number;
+  /**
+   * Constant value 'pairedVideo' indicating this is extracted from a Live Photo
+   * @platform ios
+   */
+  type: 'pairedVideo';
```

Comment:
Maybe this should be a new subtype instead?

Reply:
Yeah, if it'll be an extension of the Asset type I think this makes good sense too. So, a new property called `subtype` (that for now will only be "pairedVideo")?